### ISSUE TITLE
Konto nie przekracza limitu ilości aktywnych ofert a jednak nie udaje się publikować

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#4893]
Występuje problem: 
 - [x] z zasobem REST API
 - [x] ze środowiskiem produkcyjnym
 - [ ] ze środowiskiem testowym
 - [ ] z logowaniem lub autoryzacją
 - [ ] na portalu [https://developer.allegro.pl/](https://developer.allegro.pl/)
 - [x] inny

Po nieudanej próbie publikacji oferty sprawdzam przyczynę i dostaję:

_Could not change offer publication status. Nie możesz wystawiać nowych ofert - Twoje konto przekroczyło maksymalną liczbę 200000 aktywnych ofert._

Task id: 6a4d2519-dd3d-4765-946c-0139acd7a07f

A konto ma około 110 tys. ofert aktywnych. Nieaktywnych i zakończonych jest dużo mniej. Skąd zatem taka przyczyna nieudanej publikacji?

Pozdrawiam

**Comment by AureliuszBrussy** [comment]
Znamy ten błąd- pracujemy nad poprawką. Przepraszam za niedogoności. Wrócę z informacją, gdy będę wiedział o rozwiązaniu problemu.

**Comment by AureliuszBrussy** [comment]
@tomylee0 problem nie powinien już występować.

**Comment by tomylee0** [comment]
Teraz z kolei przy próbie wystawienia draftu dostaję:

_You cannot create new drafts - your account has exceeded the maximum number 20000 of drafts._

Przykładowe trace-id: 71509381d28b9def

A to nie jest prawdą. Nawet jeśli zaciąganie INACTIVE podaje wciąż zarchiwizowane oferty (#4878).

**Comment by AureliuszBrussy** [comment]
@tomylee0 tak, ten błąd nie został jeszcze rozwiązany - pracujemy nad tym.

**Comment by AureliuszBrussy** [comment]
@tomylee0 czy mógłbyś jeszcze podać świeże trace-id, gdzie próba pobrania/usunięcia szkicu kończy się błędem: "Offer not found. It was deleted, archived or never existed."?

**Comment by tomylee0** [comment]
Dziś po pobraniu draftów przez /sale/offers?limit=1000&publication.status=INACTIVE otrzymuję prawidłową listę. Tzn. wykonując GET na drafcie a potem wykonując na nim DELETE nie otrzymałem jeszcze błędu. A zrobiłem to już z ponad 1 tys. draftów. 

Przykład trace-id poprawnie wykonanej operacji GET na drafcie:
fb0c24ac1453c634

**Comment by AureliuszBrussy** [comment]
@tomylee0 tak, problem z wątku [#4878](https://github.com/allegro/allegro-api/issues/4878) oraz z tego wątku nie powinien już występować. Jeżeli trafisz jednak na jakiś przypadek - będę wdzięczny za jego wskazanie.

**Comment by tomylee0** [comment]
Wciąż jednak widzę _You cannot create new drafts - your account has exceeded the maximum number 20000 of drafts._

trace-id: 20cabd826c1d32db

**Comment by AureliuszBrussy** [comment]
@tomylee0 proszę, sprawdź teraz, problem nie powinien już wystąpić.

**Comment by tomylee0** [comment]
Generalnie wygląda na to, że teraz już działa. Znalazłem jednak kilka dziwnych przypadków na koncie które spowodowały, że mam pozostawione oferty w stanie INACTIVE co nie było moim zamiarem. Opiszę jeden przypadek. Dziś około godziny 9:08 próbowałem stworzyć draft'a przez POST https://api.allegro.pl/sale/offers. W odpowiedzi nie dostałem ani trace-id ani numeru oferty. Dostałem za to błąd 503 z treścią:

 {"code":"UC","destination":{"path":"/sale/offers","serviceTag":null,"host":"api.allegro.pl","originalDst":null},"message":"upstream connect error or disconnect/reset before headers. reset reason: connection termination","details":{"connectionId":150530702}}

Ale jednak wyszukując oferty w stanie INACTIVE znalazłem drafta który odpowiada wspomnianej przeze mnie operacji. Pomimo otrzymanego błędu draft się utworzył pod numerem 11131800401.

**Comment by AureliuszBrussy** [comment]
@tomylee0 sprawdzamy to. Jak tylko coś ustalimy - wrócę z odpowiedzią.
